proptest/src/num/float_samplers.rs

```diff
@@ -0,0 +1,338 @@
+//-
+// Copyright 2022 Michael Wright
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+//! Alternative uniform float samplers because the ones provided by the rand crate are prone
+//! to overflow. The samplers work by uniformly selecting from a set of equally spaced values in
+//! the interval and the included bounds. Selection is slighly biased towards the bounds.
```

Comment:
oh, my eyes missed this. strategy is covered, but i think an example will help as well. especially given the nuance of `Selection is slighly biased towards the bounds.` which i'm not entirely sure what that means, i would think a `Uniform` sampling would not have bias, it would be uniform across the bounds.

also `slighly` -> `slightly`